src/librbd/librbd.cc

```diff
@@ -621,6 +622,23 @@ namespace librbd {
     return 0;
   }
 
+  int RBD::features_to_string(uint64_t features, std::string *str_features)
+  {
+    *str_features = librbd::rbd_features_to_string(&features, nullptr);
+    if (features) {
```

Comment:
This is unnecessary -- just give it a std::stringstream and check it for an error message